howto-build-macos.md

```diff
@@ -83,6 +83,11 @@ If you be needed to build universal monero lib, then it will require additional
 
 If you will be needed to build monero wallet lib only for x86_64 on arm64 mac, then you need use steps above, but run build script with rosetta without arguments: `$ arch -x86_64 ./build_monero_all.sh`.
 
+Build the Decred library:
+
+`$ ./build_decred.sh`
```

Comment:
Should be in the build_monero_all.sh script.

Reply:
Okay.

Reply:
I figured it should be in the `build_all.sh` script instead.